import throwError from "..";

throwError("Error message");